.circleci/config.yml

```diff
@@ -51,6 +51,20 @@ aliases:
       path: ./src/api/log/
       destination: minitest
 
+  - &store_rspec_artifacts
```

Comment:
Would you mind adding a step to compress logs? This is annoying me for some time already...

Reply:
```
➜  Downloads ls -lah test.log                                         
Permissions Size User   Date Modified Name
.rw-r--r--  191M hvogel 22 Jun 15:41  test.log
➜ Downloads time xz -T 0 test.log
xz -T 0 test.log  41,66s user 0,45s system 711% cpu 5,917 total
➜  Downloads ls -lah test.log.xz 
Permissions Size User   Date Modified Name
.rw-r--r--  6,9M hvogel 22 Jun 15:41  test.log.xz
```

Reply:
I would prefer to create a GitHub issue for that and tackle this later.